packages/prover/modal_server.py

```diff
@@ -0,0 +1,62 @@
+import modal
+import logging
+from google.cloud.logging import Client
+from google.cloud.logging.handlers import CloudLoggingHandler
+from google.cloud.logging_v2.handlers import setup_logging
+from google.oauth2 import service_account
+
+app = modal.App("jwt-prover-v0.1.4")
+
+image = modal.Image.from_dockerfile("Dockerfile")
+
+
+@app.function(
+    image=image,
+    mounts=[
+        modal.Mount.from_local_python_packages("core"),
+    ],
+    cpu=16,
```

Comment:
You also need to specify gpu here.
Reference: https://github.com/zkemail/ether-email-auth/blob/main/packages/prover/modal_server.py#L20